registry.k8s.io/manifests/k8s-staging-etcd/promoter-manifest.yaml

```diff
@@ -8,7 +8,7 @@
 # google group for gcr.io/k8s-staging-etcd is k8s-infra-staging-etcd@kubernetes.io
 
 registries:
-- name: gcr.io/k8s-staging-etcd
+- name: gcr.io/etcd-development/etcd
```

Comment:
Question: some places are using` registry.k8s.io/etcd`, should we keep them consistent?

see https://github.com/kubernetes/kubernetes/pull/134251

Reply:
The other projects use `registry.k8s.io` as the main and, in most cases, only mirror.
There is no value in using `gcr.io/k8s-staging-*` when you already have a `gcr.io/etcd-development` with all images published there.
Obviously, you can switch to using `gcr.io/k8s-staging-etcd` at any time in the future if you decide to have a different release workflow.

Reply:
Based on the discussion in https://github.com/etcd-io/etcd/discussions/20756, we don't need to build etcd image using [cluster/images/etcd](https://github.com/kubernetes/kubernetes/tree/master/cluster/images/etcd) anymore, instead we use etcd officially released image directly. So no image will be pushed to `gcr.io/k8s-staging-etcd` any more. So we have to use `gcr.io/etcd-development/etcd` as the source here?

Reply:
OK, makes sense now. thx again for the clafication.